tomviz/DataSource.cxx

```diff
@@ -339,6 +334,10 @@ bool DataSource::reloadAndResample(int stride)
   format.setStride(stride);
   bool success = format.read(file.toLatin1().data(), image);
 
+  // If there are operators, re-run the pipeline
+  if (!operators().empty())
+    pipeline()->execute(this, operators().first());
```

Comment:
@cjh1 Does this look like a good way to re-run the pipeline?

Reply:
Yes